<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Persona_Dialog</class>
 <widget class="QDialog" name="Persona_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>287</width>
    <height>139</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Introducci√≥n de personas</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>261</width>
     <height>117</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="Nombre_label">
          <property name="text">
           <string>Nombre</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Nombre_line"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="Edad_label">
          <property name="text">
           <string>Edad</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="Edad_spinBox">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="Salario_label">
          <property name="text">
           <string>Salario</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="Salario_doubleSpinBox"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
